about.html

```diff
@@ -0,0 +1,128 @@
+    <header class="head">
+        <nav>
+            <div class="logo">
+                <a href="index.html" class="logo-nav"><img src="./assets/img/logo-navbar.png" alt="logo"></a>
+            </div>
+            <div class="nav-info">
+                <div class="menu-btn">
+                    <div class="hamburger-button"></div>
+                </div>
+                <ul class="navbar-items">
+                    <li>
+                        <a href="index.html"><i class="fa-solid fa-pizza-slice"></i>&nbsp Home</a>
+                    </li>
+                    <li>
+                        <a href="#logo" class="deals-menu"><i class="fa-solid fa-question"></i>&nbsp Why that Logo</a>
+                    </li>
+                    <li>
+                        <a href="#story" class="menu-menu"><i class="fa-solid fa-timeline"></i>&nbsp Our Story</a>
+                    </li>
+                    <li>
+                        <a href="index.html#locations" class="location-menu"><i
+                                class="fa-solid fa-location-dot"></i>&nbsp Location</a>
+                    </li>
+                    <li>
+                        <a href="#conect" class="contact-menu"><i class="fa-solid fa-phone"></i>&nbsp Contact Us</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <section class="hero">
+            <div class="hero-title">
+                <p>Made in Family...</p>
+                <h1>
+                    WHAT WE <br> ARE ABOUT
+                </h1>
+            </div>
+            <div class="hero-info">
+                <p>At Fratello Pizzas we don’t just make pizza. We make people happy. Fratello was built on the belief that sharing 
+                    pizza with family should be special, and we carry that belief into everything we do. With more than 30
+                    years of experience cooking under our belts.</p>
+            </div>
+            <div class="hero-date">
+                <h2>Do you want to know us closer? Write us directly and you can become one integrant of the Fratello family...</h2>
+                <a href="mailto:fratellopizzas.sc@gmail.com" target="_blank">fratellopizzas.sc@gmail.com</a>
+            </div>
+        </section>
+    </header>
+    <section class="fratello-logo" id="logo">
+        <h2>Fratello Pizzas Logo</h2>
+        <hr />
+        <p>The original Fratello's logo featured a semi-realistic looking an actual pizza, and the Fratello's Pizza wordmark
+            usually being placed to either side of the pizza.<br>Currently The triangles on the logo represent the brotherhood and the family.
+         Originally, Fratello's had planned to add a triangle for each new restaurant opened.
+        </p>
+        <div class="logo-middle"><img src="./assets/img/fratello-logo-negro-horizontal.png"></div>
+    </section>
+    <section class="story" id="story">
+        <h2>A quick look to our Story...</h2>
+        <hr />
```

Comment:
- [ ] This title is overlapping with the box containing the logo. Please fix that.